src/components/layouts/register/layout.styles.tsx

```diff
@@ -0,0 +1,79 @@
+import { gridBreakpoint } from "@/shared/breakpoints";
+import styled from "styled-components";
+
+export const StylesWrapper = styled.div<{}>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  .image-side {
+    position: relative;
+    width: 100vw;
+
+    div {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      text-align: center;
+      transform: translate(-50%, -50%);
+      color: white;
+      text-shadow: 1px 1px 4px #000000;
+
+      h3 {
+        font-size: 30px;
+      }
+
+      h1 {
+        font-size: 48px;
+      }
+    }
+  }
+
+  .form-wrapper > form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    h1 {
+      display: none;
+    }
+    .input-wrapper {
+      margin: 2rem 0 0 0;
+    }
+
+    .error-message {
+      color: red;
+      font-size: 0.725rem;
+      font-style: italic;
+      margin: 1rem 0;
+    }
+
+    button[type="submit"].error-hidden {
+      margin: 2rem 0;
+    }
+  }
+
+  @media (min-width: ${gridBreakpoint.lg}) {
+    flex-direction: row;
+    justify-content: center;
+    .image-side {
+      position: relative;
+      width: 50vw;
+      height: calc(100vh - 3rem);
```

Comment:
🦆 when you are using calc, you will be doing operations with external values, such as the `3rem` that you declared in this rule. However, for the outside reader programmer, this value might not mean much, since we do not know where it comes from. A good way to add a bit more info, is to turn this value into a variable and then consume it in the calc. For instance:

```
--card-width: 3rem;

calc(100vh - var(--card-width))
```

Reply:
🥔 For this one, I've added the variable of `--header-height` in globals and use that variable in the calculation to make things a bit more clear 🔥